### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/audiocraft/actions/runs/5301903502#summary-14349936077

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/audiocraft/actions/runs/5301903695#summary-14349936328

### CODE REVIEW

File: MODEL_CARD.md [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 **License** Code is released under MIT, model weights are released under CC-BY-NC 4.0.
 
-**Where to send questions or comments about the model:** Questions and comments about MusicGen can be sent via the [Github repository](https://github.com/facebookresearch/audiocraft) of the project, or by opening an issue.
+**Where to send questions or comments about the model:** Questions and comments about MusicGen can be sent via the [GitHub repository](https://github.com/facebookresearch/audiocraft) of the project, or by opening an issue.
```
Comment (jsoref) [inline MODEL_CARD.md:19]: brand

File: README.md [hunk 2]
```diff
@@ -103,7 +103,7 @@ Yes. We will soon release the training code for MusicGen and EnCodec.
 
 #### I need help for running the demo on Colab
 
-Check [@camenduru tutorial on Youtube](https://www.youtube.com/watch?v=EGfxuTy9Eeo).
+Check [@camenduru tutorial on YouTube](https://www.youtube.com/watch?v=EGfxuTy9Eeo).
```
Comment (jsoref) [inline README.md:106]: brand

**Review by jsoref — COMMENTED** [review]
Most corrections suggested by Google Sheets. All fault mine

**Review by adefossez — APPROVED** [review]

**Comment by jsoref** [comment]
@adefossez: do I need to worry about the merge conflict?

**Comment by JadeCopet** [comment]
We fixed the different misspellings ourselves as part of the larger code update for the 1.0.0 release. Sorry it more convenient to handle them like that with all the codebase changes. Thanks for the report!

**Pull request closed without merging**